3761/1/gem_statistic_fetcher.rb

```diff
@@ -0,0 +1,31 @@
+require './page_fetcher.rb'
+class GemStatisticFetcher
+  attr_reader :gem_name, :page_gem_in_ribygems, :agent, :stats
+
+  def initialize(gem_name, page_gem_in_ribygems, agent)
+    @gem_name = gem_name
+    @page_gem_in_ribygems = page_gem_in_ribygems
+    @agent = agent
+    @stats = {}
+  end
+
+  def call
+    stats = collect_statistic
+    stats[:used_by] = collect_used_by
+    stats[:gem] = gem_name
+    stats
+  end
+
+  private
+
+  def collect_statistic
```

Comment:
remove `collect_` from methods name you return `statistic` (don't like this name, but mb it's ok) and `used_by` for gem
I know it cos I in `GemStatisticFetcher` class